docs/user_guide.md

```diff
@@ -232,81 +451,214 @@ CodeChecker will get the hardcoded values for the compilers set in the
+                        (default: /home/username/.codechecker/reports)
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -t {plist}, --type {plist}, --input-format {plist}
+                        Specify the format the analysis results were created
+                        as. (default: plist)
+  --suppress SUPPRESS   Path of the suppress file to use. Records in the
+                        suppress file are used to suppress the display of
+                        certain results when parsing the analyses' report.
+                        (Reports to an analysis result can also be suppressed
+                        in the source code -- please consult the manual on how
+                        to do so.) NOTE: The suppress file relies on the "bug
+                        identifier" generated by the analyzers which is
+                        experimental, take care when relying on it.
+  --print-steps         Print the steps the analyzers took in finding the
+                        reported defect. (default: False)
+  --verbose {info,debug,debug_analyzer}
+                        Set verbosity level. (default: info)
+~~~~~~~~~~~~~~~~~~~~~
+
+For example, if the analysis was ran like:
+
+~~~~
+CodeChecker analyze ../codechecker_myProject_build.log -o "my_plists"
+~~~~
+
+then the results of the analysis can be printed with
+
+~~~~
+CodeChecker parse ./my_plists
+~~~~
+
+### _Suppress_ file
+
+~~~~~~~~~~~~~~~~~~~~~
+--suppress SUPPRESS   Path of the suppress file to use. Records in the
+                      suppress file are used to suppress the display of
+                      certain results when parsing the analyses' report.
+                      NOTE: The suppress file relies on the "bug
+                      identifier" generated by the analyzers which is
+                      experimental, take care when relying on it.
+~~~~~~~~~~~~~~~~~~~~~
+
+Certain defects reported by the analyzers might only cause clutter in the
+output, e.g. because they are false positive results, or some other reason why
+we don't want to show them, such as the bug not being relevant to your project
+as it happens in a library you use. (In case of false positives, please report
+the false positive to us and/or the LLVM community so that the checker which
+reported it could be investigated and fixed!)
+
+Suppress files contain bug hashes and suppression comments, and a suppressed
+report will not be shown in the output by default. Suppress files should not
+be edited manually, instead, a CodeChecker viewer server should manage them,
+but the suppress file could be committed into the source code repository for
+distribution to teammates.
+
+If `--suppress` is specified, `parse` will use the given suppress file and
+will not show reports on the standard output which are marked as suppressed.
+
+### Suppression in the source code
+
+Suppress comments can be used in the source files to suppress specific or all
```

Comment:
I'd rather implement this feature as part of a later pull request. This one is already huge enough.